flattentool/input.py

```diff
@@ -419,18 +444,33 @@ def read_sheets(self):
         self.workbook = openpyxl.load_workbook(self.input_name, data_only=True)
 
         self.sheet_names_map = OrderedDict((sheet_name, sheet_name) for sheet_name in self.workbook.get_sheet_names())
-
-        sheet_names = list(self.sheet_names_map.keys())
+        if self.include_sheets:
+            for sheet in list(self.sheet_names_map):
+                if sheet not in self.include_sheets:
+                    self.sheet_names_map.pop(sheet)
+        for sheet in list(self.exclude_sheets) or []:
```

Comment:
list is needed on 448 as you are not able to pop things from a map while iterating over it (in python3 as it creates a generator).

It is not needed on 451 so will remove it there.